packages/core/tests/dummy/app/controllers/pie-chart.js

```diff
@@ -4,6 +4,156 @@ import { set, get, computed, action } from '@ember/object';
 import { merge } from 'lodash-es';
 
 export default class PieChartController extends Controller {
+  apexModel = A([
+    {
+      request: {
+        metrics: [
+          {
+            metric: 'totalPageViews'
+          }
+        ],
+        logicalTable: {
+          timeGrain: 'day'
+        },
+        intervals: [
+          {
+            start: '2015-12-14 00:00:00.000',
+            end: '2015-12-15 00:00:00.000'
+          }
+        ],
+        dimensions: [
+          {
+            dimension: 'age'
+          },
+          {
+            dimension: 'browser'
+          }
+        ]
+      },
+      response: {
+        rows: [
+          {
+            dateTime: '2015-12-14 00:00:00.000',
+            'age|id': '-3',
+            'age|desc': 'All Other',
+            'browser|id': 'firefox',
+            'browser|desc': 'Mozilla Firefox',
+            uniqueIdentifier: 72620639,
+            totalPageViews: 3072620639,
+            revenue: 23435193.77284
+          },
+          {
+            dateTime: '2015-12-14 00:00:00.000',
+            'age|id': '1',
+            'age|desc': 'under 13',
+            'browser|id': 'Chrome',
+            'browser|desc': 'Google Chrome',
+            uniqueIdentifier: 55191081,
+            totalPageViews: 155191081,
+            revenue: 12498623.29348
+          },
+          {
+            dateTime: '2015-12-14 00:00:00.000',
+            'age|id': '2',
+            'age|desc': '13 - 25',
+            'browser|id': 'IE',
+            'browser|desc': 'Microsoft Internet Explorer',
+            uniqueIdentifier: 55191081,
+            totalPageViews: 3072620639,
+            revenue: 77348273.24588
+          },
+          {
+            dateTime: '2015-12-14 00:00:00.000',
+            'age|id': '3',
+            'age|desc': '25 - 35',
+            'browser|id': 'firefox',
+            'browser|desc': 'Mozilla Firefox',
+            uniqueIdentifier: 72620639,
+            totalPageViews: 72620639,
+            revenue: 98350255.98241
+          },
+          {
+            dateTime: '2015-12-14 00:00:00.000',
+            'age|id': '4',
+            'age|desc': '35 - 45',
+            'browser|id': 'Chrome',
+            'browser|desc': 'Google Chrome',
+            uniqueIdentifier: 72620639,
+            totalPageViews: 72620639,
+            revenue: 63491243.7692
+          },
+          {
+            dateTime: '2015-12-14 00:00:00.000',
+            'age|id': '4',
+            'age|desc': '35 - 45',
+            'browser|id': 'firefox',
+            'browser|desc': 'Mozilla Firefox',
+            uniqueIdentifier: 72620639,
+            totalPageViews: 72620639,
+            revenue: 35353239.99923
+          }
+        ]
+      }
+    }
+  ]);
+
+  apexOptions = {
+    series: {
+      config: {
+        colors: [
+          '#87d812',
```

Comment:
Do you think these color codes will be reused across multiple charts / files? Maybe we should store them as a const in a module where all the charts and their tests can access them.